src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -14,7 +14,7 @@
 public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
-    private Path addressBookFilePath = Paths.get("data" , "studentsdata.json");
+    private Path addressBookFilePath = Paths.get("data" , "mybookdata.json");
```

Comment:
Might want to change the name to personsdata.json so that it's easier to differentiate, as I will be adding a meetingsdata.json to store meetings. Otherwise, LGTM!